halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/account/AccountCommandProperties.java

```diff
@@ -19,4 +19,5 @@
 
 public class AccountCommandProperties {
   public static final String REQUIRED_GROUP_MEMBERSHIP_DESCRIPTION = "A user must be a member of at least one specified group in order to make changes to this account's cloud resources.";
+  public static final String BOOTSTRAP_ONLY = "A bootstrap-only account is the account in which Spinnaker itself is deployed. When true, this account will not be included the accounts managed by Spinnaker. Similarly, the bootstrap instance of Spinnaker will only contain this account. Only one account can be marked as bootstrap-only.";
```

Comment:
It feels tedious that a user has to manually flag this account after they've  (in a separate place in the config) already marked the account as being used for "bootstrapping". It might make more sense to have this command be a flag in teh `DeploymentEnvironment`, so they can say something like `hal deploy edit --account-name 'my-boostrapping-account' --bootstrap-only`

Reply:
Also it gets confusing if they've configured their k8s account with a docker registry that would have to be flagged too - which halyard should automatically address.